src/java.net.http/share/classes/jdk/internal/net/http/Exchange.java

```diff
@@ -111,8 +120,12 @@ HttpClientImpl client() {
 
     // Keeps track of the underlying connection when establishing an HTTP/2
```

Comment:
since this is now used with HTTP/3, should we update this comment?

Reply:
I have verified the other comments in this class and updated to mention or HTTP/3 in a few places in this class, in my local repo. Will appear the next time I sync this PR.

Reply:
addressed in a245450023ad189a54dfcb66a4303182fe81c972